src/main/java/org/zalando/nakadi/domain/Version.java

```diff
@@ -0,0 +1,51 @@
+package org.zalando.nakadi.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public class Version {
+    private final int major;
```

Comment:
I think it's going to help in the next story which will manipulate the version number. We already use for implementing comparison. If at the end we see that it was not useful, I can easily change it.